frontend/test/unit/specs/components/VSearchResultsGrid/v-image-cell.spec.js

```diff
@@ -0,0 +1,49 @@
+import { fireEvent } from "@testing-library/vue"
+
+import { render } from "~~/test/unit/test-utils/render"
+import { image } from "~~/test/unit/fixtures/image"
+
+import { useAnalytics } from "~/composables/use-analytics"
+
+import { IMAGE } from "~/constants/media"
+
+import VImageCell from "~/components/VSearchResultsGrid/VImageCell.vue"
+
+jest.mock("~/composables/use-analytics", () => ({
+  useAnalytics: jest.fn(),
+}))
+
+describe("VImageCell", () => {
+  let options = {}
+  let sendCustomEventMock = null
+
+  beforeEach(() => {
+    sendCustomEventMock = jest.fn()
+    useAnalytics.mockImplementation(() => ({
+      sendCustomEvent: sendCustomEventMock,
+    }))
+    options = {
+      props: {
+        image,
+        searchTerm: "cat",
+        aspectRatio: "square",
```

Comment:
Is this prop necessary for this test?

Reply:
`square` is the default for `aspectRatio`, so you're right, we can remove it.